inventory/group_vars/all.yml

```diff
@@ -64,6 +64,8 @@ postgres_encoding: en_US.utf8
 # to the defaults if present. Make sure you use the "list" notation with the dashes!
 
 default_hba_entries:
+  - {type: local, database: replication, user: "{{ db_user }}", auth_method: trust}
+  - {type: host, database: replication, user: "{{ db_user }}", address: '127.0.0.1/32', auth_method: trust}
```

Comment:
I don't understand these entries. There won't be any `replication` database in any of our servers. Aren't these the settings the new DB replica will have?

Reply:
It looks weird, but this is how it needs to be written, based on what postgres expects and how the role writes entries to the file. It's not an extra database, it's basically permission to perform replication.

Docs:
> "The value `replication` specifies that the record matches if a replication connection is requested (note that replication connections do not specify any particular database)."

https://www.postgresql.org/docs/9.5/auth-pg-hba-conf.html